fiftyone/server/aggregations.py

```diff
@@ -132,7 +132,26 @@ async def aggregate_resolver(
         awaitable=True,
     )
 
-    slice_view = view if form.mixed and "" in form.paths else None
+    slice_view = (
+        fosv.get_view(
```

Comment:
this is almost exactly the same as the call above. 1) is that right? 2) can we re-use it somehow to remain DRY?